metrics/resource_view.go

```diff
@@ -186,6 +199,11 @@ func meterExporterForResource(r *resource.Resource) *meterExporter {
 	mE.m = view.NewMeter()
 	mE.m.SetResource(r)
 	mE.m.Start()
+
+	mc := getCurMetricsConfig()
+	if mc != nil {
+		mE.m.SetReportingPeriod(mc.reportingPeriod)
```

Comment:
Is it possible to add an unit test for it?

Reply:
I thought about it.  It's hard to verify the internal of a meter.  Testing the effect(waiting for the reports) is not suitable for unit tests.